suricata.yaml.in

```diff
@@ -158,6 +158,7 @@ outputs:
       #  pipelining:
       #    enabled: yes ## set enable to yes to enable query pipelining
       #    batch-size: 10 ## number of entry to keep in buffer
+      #log-ethernet: yes
```

Comment:
I think I'd rather see this as just "ethernet" to be consistent with all the other options, ie: We have "tls", not "log-tls".